os/arch/arm/src/amebad/Kconfig

```diff
@@ -51,6 +51,49 @@ config RTL8721D_SERIAL_FIFO
 	bool "UART SERIAL FIFO"
 	default n
 
+config AMEBAD_I2S
```

Comment:
I think you should use this in Make.defs.
```
ifeq ($(CONFIG_AMEBAD_I2S),y)
CHIP_CSRCS += amebad_i2s.c
endif
```

Reply:
Ok, have added the following into Make.defs to compile amebad_i2s.c only when AMEBAD_I2S is configured